extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -147,6 +157,32 @@ export class BaseSlides extends BaseCarousel {
         this.autoplayDelay_);
   }
 
+  /**
+   * Called on user interaction to toggle the autoplay feature.
+   */
+  toggleAutoplay(toggleOn) {
+    // Set the autoPlay status to the opposite
+
+    if (toggleOn == this.shouldAutoplay_) {
+      return;
+    }
+
+    const prevAutoplayStatus = this.hasAutoplay_;
+
+    this.hasAutoplay_ = toggleOn;
+    this.shouldAutoplay_ = this.hasAutoplay_ && this.isLoopingEligible();
+
+    if (!prevAutoplayStatus && this.hasAutoplay_) {
```

Comment:
same here, should be `this.shouldAutoplay_`

Reply:
Done